### ISSUE TITLE
Exception when linting dom attributes

### ISSUE THREAD

**Issue by chrisetheridge** [pitch-io/uix#132]
**Describe the bug**

**This is happening in an unreleased version of UIx!**

The new code that lints DOM attributes throws an exception, when running on the server-side. This only seems to happen at the REPL, the code still renders correct HTML.

```clojure
1. Caused by java.lang.NullPointerException
   Cannot invoke "java.util.concurrent.Future.get()" because "fut" is null

                  core.clj: 2317  clojure.core/deref-future
                  core.clj: 2337  clojure.core/deref
                  core.clj: 2323  clojure.core/deref
           cljs_hacks.cljc:  229  shadow.build.cljs_hacks$shadow_resolve_var/invokeStatic
           cljs_hacks.cljc:  164  shadow.build.cljs_hacks$shadow_resolve_var/invoke
           cljs_hacks.cljc:  170  shadow.build.cljs_hacks$shadow_resolve_var/invokeStatic
           cljs_hacks.cljc:  164  shadow.build.cljs_hacks$shadow_resolve_var/invoke
           cljs_hacks.cljc:  168  shadow.build.cljs_hacks$shadow_resolve_var/invokeStatic
           cljs_hacks.cljc:  164  shadow.build.cljs_hacks$shadow_resolve_var/invoke
                  lib.cljc:   72  uix.lib$uix_element_QMARK_/invokeStatic
                  lib.cljc:   67  uix.lib$uix_element_QMARK_/invoke
                  lib.cljc:   90  uix.lib$normalize_element/invokeStatic
                  lib.cljc:   85  uix.lib$normalize_element/invoke
                linter.clj:   22  uix.dom.linter/eval180571/fn
                  AFn.java:  160  clojure.lang.AFn/applyToHelper
                  AFn.java:  144  clojure.lang.AFn/applyTo
                  core.clj:  669  clojure.core/apply
                  core.clj:  662  clojure.core/apply
                linter.clj:  293  uix.linter/run-linters!
                linter.clj:  291  uix.linter/run-linters!
               RestFn.java:  439  clojure.lang.RestFn/invoke
                linter.clj:  528  uix.linter/lint-element*
                linter.clj:  526  uix.linter/lint-element*
                  core.clj:  126  uix.core/invokeStatic
                  core.clj:  117  uix.core/doInvoke

```

**Include minimal repro**

Eval the following at the REPL, with version `ea747e6726e4e3b4987a4af9230635ec329de191` checked out.

```clojure
(defui foo [_]
  ($ :div#container
    ($ :p "Foo")))
```

Removing the `#container` fixes the issue.

**Expected behavior**
No exception should be thrown.

**Comment by roman01la** [comment]
@chrisetheridge #133 should fix the issue, please give it a try

**Comment by roman01la** [comment]
fixed in #133
